backend/prompt_studio/prompt_studio_core_v2/serializers.py

```diff
@@ -69,7 +73,26 @@ def to_representation(self, instance):  # type: ignore
             if prompt_instance.count() != 0:
                 for prompt in prompt_instance:
                     prompt_serializer = ToolStudioPromptSerializer(prompt)
-                    output.append(prompt_serializer.data)
+                    prompt_outputs = (
+                        PromptStudioOutputManager.objects.filter(
+                            tool_id=data.get(TSKeys.TOOL_ID),
+                            profile_manager_id=profile_manager.profile_id,
+                            prompt_id=prompt.prompt_id,
+                        )
+                        .values("prompt_id", "profile_manager_id")
+                        .annotate(document_count=Count("document_manager_id"))
+                    )
+                    coverage = {}
+                    for prompt_output in prompt_outputs:
+                        prompt_key = str(prompt_output["prompt_id"])
```

Comment:
NIT: Since these are instances of `PromptStudioOutputManager` please use the `.` (dot) notation to access the values